python/requirements_compiled.txt

```diff
@@ -14,15 +14,15 @@ absl-py==1.4.0
     #   tensorflow-metadata
     #   tensorflow-probability
 accelerate==0.28.0
-    # via -r /home/ubuntu/ray/ci/../python/requirements/ml/core-requirements.txt
+    # via -r /ray/ci/../python/requirements/ml/core-requirements.txt
```

Comment:
hmm, i used the output from [here](https://buildkite.com/ray-project/premerge/builds/27997#0190cd04-d97a-41e4-bc8c-afd5f04435af), but looks like it updated all these paths to`/ray/ci`? is this the correct way @can-anyscale ?

Reply:
hi yes this is ok; the `/home/ubuntu` was because i generated this file from my local machine and not using the one from CI